_locales/es/messages.json

```diff
@@ -1044,12 +1050,18 @@
     "receiverRcInterpolationInterval": {
         "message": "Intervalo de Interpolaci\u00f3n RC [ms]"
     },
+    "pidTuningDtermSetpointTransition": {
+        "message": "Punto de Ajuste de Transici\u00f3n de D"
+    },
     "pidTuningPtermSetpoint": {
         "message": "Punto de Ajuste de Transici\u00f3n de D"
     },
     "pidTuningDtermSetpoint": {
         "message": "Punto de Ajuste de Peso de D"
     },
+    "pidTuningDtermSetpointTransitionHelp": {
+        "message": "Con este par\u00e1metro, el Punto de Ajuste de Peso de D puede reducirse cuando las palancas est\u00e1n cerca del centro, lo cual provoca que el fin de los flips y rolls sea m\u00e1s suave.<br>El valor representa un punto del movimiento de la palanca: 0 - palanca centrada, 1 - completamente desplazada. Cuando la palanca est\u00e1 por encima de ese punto, el Punto de Ajuste de Peso se mantiene constante constante en el valor configurado. Cuando la palanca se coloca debajo de ese punto, el Punto de Ajuste de Peso se reduce proporcionalmente, llegando a 0 en la posici\u00f3n central de la palanca. <br> El valor de 1 da el m\u00e1ximo efecto de suavizado, mientras que el valor de 0 mantiene el Punto de Ajuste de Peso fijo en su valor configurado sobre el total rango de la palanca."
+    },
     "pidTuningPtermSetpointHelp": {
```

Comment:
This should be removed, it's not used any more, and was removed from the English file.

Reply:
I let them because I wasn't too sure in which commit will the translations be included. If they're not needed I can deprecate them and remove all the deprecated strings.